addons/stacks/src/commands/actions/deploy_contract.rs

```diff
@@ -148,48 +165,56 @@ lazy_static! {
                             documentation: "Type of transform (supported: 'contract_source_find_and_replace').".into(),
                             typing: Type::string(),
                             optional: false,
-                            interpolable: true,
+                            tainting: true,
+                            internal: false,
                         },
                         ObjectProperty {
                             name: "from".into(),
                             documentation: "The pattern to locate.".into(),
                             typing: Type::string(),
                             optional: false,
-                            interpolable: true,
+                            tainting: true,
+                            internal: false,
                         },
                         ObjectProperty {
                             name: "to".into(),
                             documentation: "The update.".into(),
                             typing: Type::string(),
                             optional: false,
-                            interpolable: true,
+                            tainting: true,
+                            internal: false,
                         }, ])),
                     optional: true,
-                    interpolable: true
+                    tainting: true,
+                    internal: false
                 },
                 contracts_ids_dependencies: {
                     documentation: "Contracts that are depending on this contract at their deployment.",
                     typing: Type::array(Type::string()),
                     optional: true,
-                    interpolable: true
+                    tainting: true,
+                    internal: false
                 },
                 contracts_ids_lazy_dependencies: {
```

Comment:
I think we need a better name for this field and the above `contracts_ids_dependencies`. It's weird that all three words are plural.

`dependency_contract_ids` & `lazy_dependency_contract_ids`? Is "lazy dependency" a known concept? I've never heard of that before